src/utils/provider.ts

```diff
@@ -143,11 +224,26 @@ export class Block {
     }
   }
 
+  /**
+   * Create a Block instance
+   * @param {BlockIdentifier} _identifier  hex string and BigInt are detected as block hashes.
+   * decimal string and number are detected as block numbers.
+   * text string are detected as block tag.
+   * null is considered as a 'pending' block tag.
+   */
   constructor(_identifier: BlockIdentifier) {
     this.setIdentifier(_identifier);
   }
 
   // TODO: fix any
+  /**
+   * Return the identifier as a string
+   * @example
+   * ```typescript
+   * const result = new provider.Block(123456n).queryIdentifier;
+   * // result = "blockHash=0x1e240"
+   * ```
+   */
   get queryIdentifier(): any {
```

Comment:
Do we need @returns on these?

Reply:
Not really a documented case...
@type is rarely used with Typescript. Mainly for JS when unclear type response ; not applicable for this case.
Seems that most of projects are just implementing a comment, and sometime an example, but nothing else: no @returns.
Well, to be more royalist than the king, I will add @returns...